internal/pkg/agent/transpiler/vars_test.go

```diff
@@ -79,6 +79,12 @@ func TestVars_Replace(t *testing.T) {
 			false,
 			false,
 		},
+		{
+			"${un-der_score.missing|un-der_score.missing2|data}",
```

Comment:
Please add a comment that `data` will be resolved as `other.data` since that is set as default provider